From d094481c76ede7990e2860ec8b5085ee18e29760 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Thu, 28 Jan 2021 15:51:14 +0200
Subject: [PATCH] [SW-20154] gaudi2_arc: change the base address of sram/spsram

Change the base addresses of SRAM and SPSRAM to the
physical address bases, since they are used by the
DMA

Change-Id: Ief8ca43804c3169313c12f23679d52208e004b76
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2_arc/soc.h b/gaudi2_arc/soc.h
index 964878e..12ac0cb 100644
--- a/gaudi2_arc/soc.h
+++ b/gaudi2_arc/soc.h
@@ -17,8 +17,8 @@
 
 #define SPI_FLASH_BLOCK_SIZE			1024
 
-#define HL_SRAM_BASE				0x7FF0000000ULL
-#define HL_SP_SRAM_BASE            		0x7FFBFE0000ULL
+#define HL_SRAM_BASE				0x1000007FF0000000ULL
+#define HL_SP_SRAM_BASE            		0x1000007FF7FE0000ULL
 
 /*
  * This is not SDRAM Base, but the base address of General region
